lua/rose-pine/theme.lua

```diff
@@ -410,6 +410,49 @@ local theme = {
 	VimwikiLink = { fg = p.rose },
 	VimwikiList = { fg = p.iris },
 	VimwikiNoExistsLink = { fg = p.love },
+
+	-- LspSaga
+	-- https://github.com/tami5/lspsaga.nvim (fork of https://github.com/glepnir/lspsaga.nvim)
+	DiagnosticInformation = { link = 'DiagnosticInfo' },
+	DiagnosticWarning = { link = 'DiagnosticWarn' },
+
+	LspSagaCodeActionTitle = {
+		fg = p.rose,
+		style = 'bold',
+	},
+	LspSagaCodeActionBorder = { fg = p.subtle },
+	LspSagaCodeActionTruncateLine = { link = 'LspSagaCodeActionBorder' },
+	LspSagaCodeActionContent = { fg = p.foam },
+	LspSagaDiagnosticBorder = { fg = p.subtle },
+	LspSagaDiagnosticHeader = {
+		fg = p.foam,
+		style = 'bold',
+	},
+	LspSagaDiagnosticTruncateLine = { link = 'LspSagaDiagnosticBorder' },
+	LspSagaFinderSelection = { fg = p.gold },
+	LspSagaLspFinderBorder = { fg = p.subtle },
+    LspSagaAutoPreview = { fg = p.subtle },
+    LspSagaDefPreviewBorder = { fg = p.subtle },
+	DefinitionIcon = { fg = p.rose },
+    DefinitionCount = { fg = p.rose },
+    ReferencesCount = { fg = p.rose },
+    ReferencesIcon = { fg = p.rose },
+    DefinitionPreviewTitle = {
+        fg = p.rose,
+        style = "bold",
+    },
```

Comment:
Are these groups provided by LspSaga? If not, they should be moved to a more generic location.

Reply:
Yes, these are provided by LspSaga and not used anywhere else.